packages/core/src/shared/ui/sam/ecr.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Same as previous comment for naming, maybe change file name to `ecr-prompter`

Reply:
Rename file to have **Prompter** suffix